src/nvim/vvars.lua

```diff
@@ -229,6 +232,7 @@ M.vars = {
     ]=],
   },
   ['false'] = {
+    type = 'boolean',
```

Comment:
This isn't actually a Boolean value.

Reply:
`type(vim.v['false'])` gives boolean and `vim.v['false'] == false`. Could you please elaborate?

Reply:
Hmm I must be mistaken.